.gitignore

```diff
@@ -3,3 +3,7 @@
 /build/
 node_modules/
 tscommand*.txt
+
+# vim files
```

Comment:
these should be added to a global gitignore file, not each project where you use vim